src/components/common/Dropdown/index.tsx

```diff
@@ -0,0 +1,45 @@
+import { memo } from 'react';
+
+import { Icons } from '@/assets/icons';
+import { useDropdown } from '@/hooks/common/useDropdown';
+
+import { Props } from './Dropdown.types';
+import { DropdownVariants } from './Dropdown.variants';
+
+export const Dropdown = memo(({ selectedCategory, onSelectCategory, className }: Props) => {
+  const { isOpen, handleToggle, handleSelect, categories } = useDropdown(onSelectCategory);
+
+  const ArrowDownIcon = Icons.arrowDown;
+  const ArrowUpIcon = Icons.arrowUp;
+
+  return (
+    <div className={`relative inline-block ${className}`}>
+      <button
+        type="button"
+        className={DropdownVariants({ state: 'default' })}
+        onClick={handleToggle}
+      >
+        <div className="flex items-center">
+          {selectedCategory}
+          <span className="ml-2">
+            {isOpen ? <ArrowUpIcon className="inline" /> : <ArrowDownIcon className="inline" />}
```

Comment:
`IconButton` 컴포넌트가 존재해서 변경해도 좋을 것 같아요~
```suggestion
<IconButton name={isOpen ? 'arrowUp' : 'arrowDown'}  /> 
```